buildstockbatch/gcp/gcp.py

```diff
@@ -355,10 +355,16 @@ def list_jobs(self):
         logger.info(f'Showing the first 10 existing jobs that match: {request.filter}\n')
         response = client.list_jobs(request)
         for job in response.jobs:
+            task_counts = collections.defaultdict(int)
             logger.debug(job)
```

Comment:
What does this debug line output — is it redundant with the info logs (is it needed)?

Reply:
It prints out the long version of the job information with lots more details (but only when the `--verbose` flag is set).

Reply:
Is it redundant with the 'info' logging that follows? Would it make sense to not do the info logging if debug logging is enabled?